README.md

```diff
@@ -61,6 +61,60 @@ If you have a preferred import path to suggest, append the suggestion after a `=
 -paths "fmt.{Errorf}=github.com/pkg/errors.{Errorf}"
 ```
 
+### Ignoring problems
+
+If you want to ignore a problem reported by `faillint` you can add a linter directive.
+
+#### Line-based linter directives
+
+Line-based linter directives can be applied to imports or functions you want to tolerate.  The format is,
+
+```go
+//faillint:ignore reason
+```
+
+For example,
+
+```go
+package a
+
+import (
+        //faillint:ignore Whatever your reason is.
+        "errors"
+        "fmt" //faillint:ignore Whatever your reason is.
+)
+
+func foo() error {
+        //faillint:ignore Whatever your reason is.
+        return errors.New("bar!")
+}
+```
+
+#### File-based linter directives
+
+File-based linter directives can be applied to ignore `faillint` problems in a whole file.  The format is,
+
+```go
+//faillint:file-ignore reason
+```
+
+This must be placed in the `package` docs and, conventionally, these comments should be placed near the top of the file.
```

Comment:
I think people might get confused on what `package docs` means here. Maybe writing it simpler would be better, such as `This must be placed above the package keyword, near the top of the file`. What do you think?

Reply:
Problem was it did have to be attached to the `package` comment/docs.  However, I agree that's confusing and probably not really correct/ideal.  I'll loosen it to search all comments in the file and then the phrase can be, "This may be placed anywhere in the file but conventionally it should be placed at or near the top of the file."

Reply:
Done in https://github.com/fatih/faillint/pull/11/commits/868a3ea1c233556d536f5ebed54f8378a368dcb5.